README.md

```diff
@@ -55,8 +55,7 @@ Functional tests not originating from the openjdk regression suite, that include
 --- 
 
 ##### jck tests - Compliance tests
-TCK tests (under the OpenJDK Community TCK License Agreement), in compliance with the license agreement.  While this test material is currently not run at the Adoptium project (see the [support statement](https://adoptopenjdk.net/support.html#jck) for details), those with their own OCTLA agreements may use the Adoptium test automation infrastructure to execute their TCK test material in their own private Jenkins servers.
-
+TCK tests (under the OpenJDK Community TCK License Agreement), in compliance with the license agreement. While this test material is currently not run at the Adoptium project (see the support statement for details), those with their OCTLA agreements may use the Adoptium test automation infrastructure to execute their TCK test material on their own private Jenkins servers.
```

Comment:
This paragraph has not been improved with this change, the main change here is removing the hyperlink.  

Since this paragraph is actually no longer true, now that we do run TCKs at the Adoptium project under the Temurin-compliance subproject, I will recommend a core committer needs to make these changes to ensure the information is accurate (which will be done under issue: https://github.com/adoptium/aqa-tests/issues/1558).